api/tasks.go

```diff
@@ -132,6 +132,17 @@ func (r *ReschedulePolicy) Copy() *ReschedulePolicy {
 	return nrp
 }
 
+func (p *ReschedulePolicy) String() string {
+	if p == nil {
+		return ""
+	}
+	if *p.Unlimited {
+		return fmt.Sprintf("unlimited with %v delay, max_delay = %v", *p.DelayFunction, *p.MaxDelay)
+	} else {
+		return fmt.Sprintf("%v in %v with %v delay, max_delay = %v", *p.Attempts, *p.Interval, *p.DelayFunction, *p.MaxDelay)
```

Comment:
@preetapan your PR comment had "delay" not "max_delay" for the non-unlimited case -- just checking that was simply a typo?

Reply:
yeah i meant max_delay. fixed comment